chapter2/tests/t20/terraform/dev/_rds.tf

```diff
@@ -0,0 +1,53 @@
+module "db" {
+  source  = "terraform-aws-modules/rds/aws"
+  identifier            = local.labels.rds
+  engine                = "mysql"
+  family                = "mysql8.0"
+  major_engine_version  = "8.0"
+  engine_version        = "8.0"
+  instance_class        = "db.t4g.micro"
+  allocated_storage     = 20
+  max_allocated_storage = 1000
+  skip_final_snapshot   = true
+
+  db_name                = var.rds_db_name
+  username               = var.rds_db_user
+  create_random_password = false
+  password               = random_password.db_password.result
```

Comment:
use keepers for the password to avoid mutation.

Reply:
> use keepers for the password to avoid mutation.

@dmitry-mightydevops
Done